### DISCUSSION TITLE
arbitrary lagrangian eulerian (ALE)

### DISCUSSION THREAD

**Discussion by XingjunFang** [mfem/mfem#3890] [Q&A]
Hi, 

I am working on fluid simulation using the arbitrary lagrangian eulerian (ALE) approach. I am comfortable with the miniapp/navier, but not sure how to modify the coordinates/nodes and update the FEM space afterwards. Could you please give some suggestions?

Thank you

**Comment by vladotomov** [comment]
@XingjunFang take a look at `miniapps/pmesh-optimizer` and [_`Remhos`_](https://github.com/CEED/Remhos). Alternative way to remap is `miniapps/meshing/gslib/field-interp.cpp`.
